### ISSUE TITLE
Changeset Summary: imagery_used / local GPX

### ISSUE THREAD

**Issue by ax3l** [openstreetmap/iD#2804]
The change set summary (`https://www.openstreetmap.org/changeset/<ID>#map=<mapID>`) using the default online editor iD only shows `;Local GPX` in `imagery_used` when in the same edit also traces of others were used (aka `;OpenStreetMap GPS traces`).

Nevertheless, when adding new ways using only "local GPX" (which is btw. also shown for *uploaded* own, *trackable* GPX) is sufficient and should be reflected correctly in `imagery_used`.

**Comment by bhousel** [comment]
I'm having trouble reproducing this issue.  I do see "OpenStreetMap GPS traces" in the `imagery_used` changeset tag of a recent edit - although this only appears if the GPS trace overlay is active while I make my edit.   

This means: if you just enable the layer and disable it, or enable it after your edits are already stored in the history, it doesn't go into `imagery_used`.  It needs to be actually visible active while editing.

**Comment by ax3l** [comment]
> I'm having trouble reproducing this issue.

I did the following steps:
  on `openstreetmaps.org` I went to `GPS Traces` -> `See your traces`, selected one and clicked `edit` to show it on the map.

The single trace (but not other contributed traces from other users) is now visible in OSM.

Now do some edit and save immediately without changing overlays, etc.
`imagery_used` will then *not* reflect `Local GPX` even if it was used this way.
